angular-app/src/app/models/interfaces/IAppointment.ts

```diff
@@ -11,5 +11,5 @@ export interface IAppointment {
     cancelled?: boolean;
     totalSlots?: number;
     tags?: string[];
-    participants: string[];
-}
\ No newline at end of file
+    participants: string[]| IUser[];
+}
```

Comment:
I've removed `IUser[]` as being a possible type for participants since we won't be storing whole `User` objects in the database and our backend dto has it as just a string array.